src/transferlistwidget.cpp

```diff
@@ -741,6 +754,10 @@ void TransferListWidget::displayListMenu(const QPoint&)
     connect(&actionForce_recheck, SIGNAL(triggered()), this, SLOT(recheckSelectedTorrents()));
     QAction actionCopy_magnet_link(QIcon(":/Icons/magnet.png"), tr("Copy magnet link"), 0);
     connect(&actionCopy_magnet_link, SIGNAL(triggered()), this, SLOT(copySelectedMagnetURIs()));
+
+    QAction actionCopy_caption_list(tr("Copy name"), 0);
```

Comment:
Can you add a screenshot here? Just the part of the menu where you see the results of your modified code.